src/Node.js

```diff
@@ -0,0 +1,133 @@
+  // ### DOM Node Shim
+  //
+  // This represents a superficial form of a DOM node which contains most of
+  // the properties that a regular DOM node would contain, but only the ones
+  // needed by slowparse.
+module.exports = (function(){
+  "use strict";
+
+  function Node(nodeValue, namespaceURI) {
+    var self = this;
+    self.nodeValue = nodeValue || "";
+    self.namespaceURI = namespaceURI || "";
+    self.parentNode = false;
+    self.nextSibling = false;
+    self.isVoid = false;
+    self.childNodes = [];
+    self.attributes = [];
+    Object.defineProperties(self, {
```

Comment:
We're *not* writing a real Node implementation here, this code is arguably correct, but also way more than we need: Slowparse is never going to call `textnode.innerHTML`, it's only going to call that on nodes that it knows should support `innerHTML`, so there is no reason to do these checks in the slightest.